.github/workflows/scripts/spdx-tools-java-wrapper.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: Maximilian Huber <maximilian.huber@tngtech.com>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+set -euo pipefail
+
+version="1.1.3"
+url="https://github.com/spdx/tools-java/releases/download/v${version}/tools-java-${version}.zip"
+jar="$HOME/spdx-tools-java/tools-java-${version}-jar-with-dependencies.jar"
+
+bootstrap() {
+    if [[ ! -f "$jar" ]]; then
+        mkdir -p "$(dirname "$jar")"
+        local tmpdir="$(mktemp -d)"
+        (cd "$tmpdir"
+        wget -q "$url"
+        unzip "tools-java-${version}.zip"
```

Comment:
Maybe use `-d extdir` instead of moving the file below?

Reply:
there are several jar files, and I just wanted to keep one

Reply:
Then you could just name the file to extract and pass it to `unzip`.